extensions/kubernetes-config/runtime/src/main/java/io/quarkus/kubernetes/client/runtime/AbstractKubernetesConfigSourceUtil.java

```diff
@@ -79,6 +81,17 @@ abstract ConfigSource createPropertiesConfigSource(String kubernetesConfigSource
         return result;
     }
 
+    private int getOrdinal(int ordinalOffset) {
+        final OrdinalData ordinalData = ordinalData();
+        /*
+         * We don't want a large list of sources to cause an "overflow" into an Ordinal of another ConfigSource
+         * so we just let the last ones all use the max ordinal
+         * this is not fool proof, but it's very unlikely that an application will need to define
+         * a list with 10+ sources...
```

Comment:
Haha, I was thinking about this too :-) I think you're right here.

Reply:
Glad to hear it :)